### ISSUE TITLE
NTR: CT: jugular foramen stenosis (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8966]
New term request CT: jugular foramen stenosis (elthat_120822075952):   UMLS: 2674495

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000376
- label: Jugular foramen stenosis
- parents: [Abnormal skull base morphology (HP:0002693)](https://hpo.jax.org/app/browse/term/HP:0002693)
- pmid: PMID:17236138
- def: Reduction in diameter of the jugular foramen, an opening in the base of the skull that is located behind the carotid canal and is formed by the temporal bone and the occipital bone. The inferior petrosal sinus, three cranial nerves, the sigmoid sinus, and meningeal arteries pass through the jugular foramen.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8966
- synonyms:
